ui/src/components/applications/Applications.vue

```diff
@@ -240,7 +240,7 @@ export default {
     }, 150);
 
     // get list of installed apps
-    context.getHideUninstall();
+    context.getApps();
```

Comment:
we have to keep this part, it is used to show the dotted menu

Reply:
the function getHideUninstall is back